packages/wonder-blocks-dropdown/src/components/dropdown-opener.tsx

```diff
@@ -51,15 +51,21 @@ type Props = Partial<Omit<AriaProps, "aria-disabled">> & {
      * If the dropdown has an error.
      */
     error?: boolean;
+    /**
+     * The role of the opener.
+     */
+    role: "combobox" | "button";
 };
 
 type DefaultProps = {
     disabled: Props["disabled"];
+    role: Props["role"];
 };
 
 class DropdownOpener extends React.Component<Props> {
     static defaultProps: DefaultProps = {
         disabled: false,
+        role: "combobox",
```

Comment:
Since the `role` prop is required, we don't need to provide it in the default props!